roles/mcpserver-model/templates/model.deployment.yaml.j2

```diff
@@ -0,0 +1,215 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    {{ lookup("template", "../common/templates/labels/common.yaml.j2")  | indent(width=4) | trim }}
+    {{ lookup("template", "../common/templates/labels/version.yaml.j2") | indent(width=4) | trim }}
+    app.kubernetes.io/component: '{{ deployment_type }}-api'
```

Comment:
IDK if you'd want to change references of `api` to `mcp`.

Using `api`does no harm; it's just misleading.

Reply:
I will drop `-api`.